src/balancerd/src/lib.rs

```
@@ -196,63 +199,90 @@ impl BalancerService {
-                        ld::ContextBuilder::new(format!(
-                            "{}/{}/{}",
-                            provider, region, cfg.build_version
-                        ))
-                        .kind("balancer")
-                        .set_string("provider", provider)
-                        .set_string("region", region)
-                        .set_string("version", cfg.build_version.to_string())
-                        .build()
-                        .map_err(|e| anyhow::anyhow!(e))?,
-                    );
-                } else {
-                    builder.add_context(
-                        ld::ContextBuilder::new(format!(
-                            "{}/{}/{}",
-                            "unknown", region, cfg.build_version
-                        ))
-                        .anonymous(true) // exclude this user from the dashboard
-                        .kind("balancer")
-                        .set_string("provider", "unknown")
-                        .set_string("region", region)
-                        .set_string("version", cfg.build_version.to_string())
-                        .build()
-                        .map_err(|e| anyhow::anyhow!(e))?,
-                    );
-                }
-                Ok(())
-            },
+        // Configure dyncfg sync
+        match (
             cfg.launchdarkly_sdk_key.as_deref(),
-            cfg.config_sync_timeout,
-            cfg.config_sync_loop_interval,
-            move |updates, configs| {
-                if has_tracing_config_update(updates) {
-                    match tracing_config(configs) {
-                        Ok(parameters) => parameters.apply(&tracing_handle),
-                        Err(err) => warn!("unable to update tracing: {err}"),
-                    }
-                }
-            },
-        )
-        .await
-        {
-            // Log but continue anyway. If LD is down we have no way of fetching the previous value
```

We should preserve the justification for ignoring errors here